README.md

```diff
@@ -2,85 +2,156 @@
 
 ## [Swiss pairings](https://en.wikipedia.org/wiki/Swiss-system_tournament)
 
-`/tournament start swiss:2:PT65M:4 "Netrunner Draft #5" @Alice @Bob Carol @Dave @Eve Frank Grace`
-> Tournament `T1` started.
+`/tournaments`
+> Available commands:
+* game
+* round
+* score
+* tournament
+
+`/tournament`
+> Available `/tournament` sub-commands:
+* create
+* set
+    * title
+    * players
+* use
+    * swiss
+* show
+* start
+
+`/tournament create`
+> Tournament `T1` created.
+
+`/tournament set title Netrunner Draft #5`
+> Tournament `T1` title set.
+
+`/tournament set players Alice Bob Carol Dave Eve Frank Grace`
+> Tournament `T1` players set.
 
 `/tournament show`
-> Tournament `T1` “Netrunner Draft #5” has started on 2016-05-16 17:35.
+> Tournament `T1` "Netrunner Draft #5" has not started yet.
 
-> It uses Swiss pairings. Number of rounds: 2. Time for each round: 65 minutes. Byes are worth 4 points.
+> Participants: Alice, Bob, Carol, Dave, Eve, Frank, Grace.
+
+> It uses no system.
+
+> It is the current tournament.
+
+`/tournament use swiss`
+> Tournament `T1` set to use Swiss Pairings.
+
+`/tournament show`
+> Tournament `T1` "Netrunner Draft #5" has not started yet.
+
+> Participants: Alice, Bob, Carol, Dave, Eve, Frank, Grace.
+
+> It uses Swiss pairings. Number of rounds: 3. Time for each round: 30 minutes. Byes are worth 2 points.
+
+> It is the current tournament.
+
+`/tournament`
+> Available `/tournament` sub-commands:
+* create
+* set
+    * title
+    * players
+    * rounds
+    * round-time
+    * bye-points
+* use
+    * swiss
+* show
+* start
+
+`/tournament set rounds 2`
+> Tournament `T1` number of rounds set.
+
+`/tournament set round-time PT65M`
```

Comment:
[The only legit one](https://en.wikipedia.org/wiki/ISO_8601#Durations).
It will be documented in the "Available sub-commands" responses, when I find the time to finish the example.